api/napi/src/interpreter/component_instance.rs

```diff
@@ -65,6 +65,55 @@ impl JsComponentInstance {
         Ok(())
     }
 
+    #[napi]
+    pub fn get_global_property(
+        &self,
+        env: Env,
+        global_name: String,
+        name: String,
+    ) -> Result<JsUnknown> {
+        if !self.definition().globals().contains(&global_name) {
+            return Err(napi::Error::from_reason(format!("Global {global_name} not found")));
+        }
+        let value = self
+            .inner
+            .get_global_property(global_name.as_ref(), name.as_ref())
+            .map_err(|e| Error::from_reason(e.to_string()))?;
+        super::value::to_js_unknown(&env, &value)
+    }
+
+    #[napi]
+    pub fn set_global_property(
+        &self,
+        env: Env,
+        global_name: String,
+        prop_name: String,
+        js_value: JsUnknown,
+    ) -> Result<()> {
+        let ty = self
+            .inner
+            .definition()
+            .global_properties_and_callbacks(global_name.as_str())
+            .ok_or(napi::Error::from_reason(format!("Global {global_name} not found")))?
+            .find_map(|(name, proptype)| if name == prop_name { Some(proptype) } else { None })
+            .ok_or(())
+            .map_err(|_| {
+                napi::Error::from_reason(format!(
+                    "Property {prop_name} of global {global_name} not found in the component"
```

Comment:
Please add a test that covers this code path (you can fold that into the existing test).